pkg/agent/agent.go

```diff
@@ -449,6 +454,34 @@ func (k *Klocksmith) waitForPodDeletion(pod v1.Pod) error {
 	})
 }
 
+// wait until a pod matching the provided label is running on this node
+func (k *Klocksmith) waitForPodWithLabel() error {
+	if k.waitForPodLabel == "" {
+		// No label specified, nothing to do
+		return nil
+	}
+
+	for {
+		glog.Infof("Waiting for pod running on this node matching label: %s", k.waitForPodLabel)
+		time.Sleep(30 * time.Second)
```

Comment:
can you make this duration a const  so we can easily change/tune latter if we need too.

Reply:
Good catch, I just noticed there was already a const at the top of the file called `defaultPollInterval`, so I changed it to use that.